src/Router.php

```diff
@@ -25,6 +25,7 @@ class Router {
     string $page,
```

Comment:
I noticed that the title of this function is mispelled. Can you change it to `genRouteModal`?